docs/blog/posts/2024/srlinux-asymmetric-routing.md

```diff
@@ -0,0 +1,1801 @@
+---
+date: 2024-10-14
+tags:
+    - bgp
+    - evpn
+authors:
+    - aninda
+---
+
+# Asymmetric routing with SR Linux in EVPN VXLAN fabrics
+
+This post dives deeper into the asymmetric routing model on SR Linux.  
```

Comment:
comment from Jorge

In the intro, "bridge-route-bridge model, implying that the ingress leaf bridges the packet into the Layer 2 domain, routes it from one VLAN/VNI to another and then bridges the packet across the VXLAN fabric to the destination." --> asymmetric really implies a different number and type of lookups on ingress leaf when compared to egress leaf. The former doing mac-lookup, ip-lookup, mac-lookup and the latter (egress leaf) just doing mac-lookup. I think you mean the same, but it would be good to elaborate.
also please refer to RFC9135 which is the RFC that introduces the concept of asymetric and symmetric

Reply:
added clarification.